pinot-core/src/main/java/com/linkedin/pinot/core/data/partition/ByteArrayPartitionFunction.java

```diff
@@ -21,33 +21,36 @@
 
 /**
  * Implementation of {@link Byte array partitioner}
- * {@link org.apache.kafka.producer.ByteArrayPartitioner}.
  *
  */
 public class ByteArrayPartitionFunction implements PartitionFunction {
```

Comment:
Since this class is used only for testing, I would suggest moving it to some test utils. I thought I had already asked @rven1 to do this, but then maybe I missed.